lib/opengl32-sys/src/lib.rs

```diff
@@ -11,8 +11,8 @@ extern "system" {
     // pub fn GlmfEndPlayback();
     // pub fn GlmfInitPlayback();
     // pub fn GlmfPlayGlsRecord();
-    // pub fn glAccum();
-    // pub fn glAlphaFunc();
+    pub fn glAccum(op: GLenum, value: GLfloat);
+    // pub fn glAlphaFunc(func: GLenum, ref: GLclampf);
```

Comment:
I ran into trouble here, since ref is a reserved keyword. Is there a way to get around this and, if not, can I change this to _ref or reference and still have the FFI binding work?